ramls/configuration.json

```diff
@@ -27,6 +27,10 @@
       "description": "Encryption key for encoding/decoding profile picture",
       "type": "string"
     },
+    "maxFileSize": {
+      "description": "Size of profile picture",
+      "type": "number"
+    },
```

Comment:
Please add about this config parameter in the readme.md file.

Reply:
fixed